test/development/app-dir/error-overlay/error-ignored-frames/error-ignored-frames.test.ts

```diff
@@ -28,27 +20,11 @@ describe('error-ignored-frames', () => {
     await toggleCollapseCallStackFrames(browser)
 
     const expendedStack = await getStackFramesContent(browser)
-    if (isTurbopack) {
-      expect(expendedStack).toMatchInlineSnapshot(`
-       "at Page (app/page.tsx (2:9))
-       at resolveErrorDev ()
-       at processFullStringRow ()
-       at processFullBinaryRow ()
-       at progress ()
-       at InnerLayoutRouter ()
-       at OuterLayoutRouter ()"
-      `)
-    } else {
-      expect(expendedStack).toMatchInlineSnapshot(`
-       "at Page (app/page.tsx (2:9))
-       at resolveErrorDev ()
-       at processFullStringRow ()
-       at processFullBinaryRow ()
-       at progress ()
-       at InnerLayoutRouter (../src/client/components/layout-router.tsx (415:5))
-       at OuterLayoutRouter (../src/client/components/layout-router.tsx (615:19))"
-      `)
-    }
+    const ignoreListedStack = expendedStack.replace(defaultStack, '')
+    // We don't care about the exact stack trace that was ignore-listed.
+    // It'll contain implementation details that may change and
+    // shouldn't break this test.
+    expect(ignoreListedStack.trim()).toMatch(/at .*/)
```

Comment:
This stops working once you uncomment `toggleCollapseCallStackFrames` i.e. the feature stops working. The `expendedStack` would be equal to the `defaultStack` and therefore  `ignoreListedStack` would be empty failing the assertion.